phalcon/cache/backend/redis.zep

```diff
@@ -71,6 +71,10 @@ class Redis extends Backend implements BackendInterface
 			let options = [];
 		}
 
+		if isset options["redis"] && !empty options["redis"] && typeof options["redis"] == "object" {
```

Comment:
is use isset require? in PHP - empty already include isset - so can delete isset, but stay `!empty`
(i'm PHP developer and can mistake in Zephir)